src/logion/controllers/locrequest.controller.ts

```diff
@@ -191,7 +191,7 @@ export class LocRequestController extends ApiController {
         }
         await this.locRequestService.addNewRequest(request);
         const { userIdentity, userPostalAddress, identityLocId } = await this.locRequestAdapter.findUserPrivateData(request);
-        if (request.status === "REQUESTED" && !accountEquals(authenticatedUser, owner)) {
+        if (request.status === "REVIEW_PENDING" && !accountEquals(authenticatedUser, owner)) {
```

Comment:
The second term of the condition is useless now.

Reply:
I think it's still useful. There are cases where LOC is created by the LLO (i.e. Logion identity)

Reply:
...but they should not be REVIEW_PENDING then, right?

Reply:
Exact.